client/src/main/java/se/fortnox/reactivewizard/client/ConcatenatedJsonObjectsDeserializer.java

```diff
@@ -0,0 +1,123 @@
+package se.fortnox.reactivewizard.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.Sinks;
+import se.fortnox.reactivewizard.util.ReflectionUtil;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class ConcatenatedJsonObjectsDeserializer {
```

Comment:
Have we looked at using https://www.javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.14.2/com/fasterxml/jackson/core/JsonParser.html?

Reply:
I did. Looking at [JsonArrayDeserializer](https://github.com/FortnoxAB/reactive-wizard/blob/master/client/src/main/java/se/fortnox/reactivewizard/client/JsonArrayDeserializer.java), I thought it may be easier to just write the deserializer without using JsonParser :)